src/js/components/Select/Select.js

```diff
@@ -284,6 +285,11 @@ const Select = forwardRef(
 
     const iconColor = getIconColor(theme);
 
+    let ariaLabelledBy;
+    if (formContext?.useFormField({})?.inForm && id && !ariaLabel) {
```

Comment:
I think we wanted to check if there was a placeholder

Reply:
added